Makefile

```diff
@@ -17,8 +17,10 @@
 
 .PHONY: help clean clean-dist build dev test system-test test-travis release pip-release bin-release dev-binder .binder-image audit audit-licenses
 
-BASE_VERSION=0.1.0.dev9
-VERSION=$(BASE_VERSION)-incubating
+BASE_VERSION=0.1.0
+DEV_VERSION?=dev9
+PYPI_VERSION=$(BASE_VERSION).$(DEV_VERSION)
```

Comment:
This is problematic for when we are ready to release. It is always placing the `.`